src/d_deh.c

```diff
@@ -1975,6 +1976,9 @@ void deh_procThing(DEHFILE *fpin, FILE* fpout, char *line)
                   // Don't worry about conversion -- simply print values
                   if (fpout) fprintf(fpout, "Bits = 0x%08lX = %ld \n",
                                      value, value);
+
```

Comment:
Should we also consider the case that the translucency flag is removed from the map object?

Reply:
Something like this?
```diff
if (value & MF_TRANSLUCENT ||
    mobjinfo[indexnum].flags & MF_TRANSLUCENT)
  D_SetTranslucency(indexnum);
```
Should I apply reformatting from above?

Reply:
> Something like this?

Yes, but before applying the change, if course. 😉 

> Should I apply reformatting from above?

Let's first get the logic right.

Reply:
> Let's first get the logic right.

I think we have reached this point now. 👍